scripts/create-package.js

```diff
@@ -101,7 +90,7 @@ function handleError(error, errorPrependMessage) {
 
 function performStep(stepIndex) {
   if (stepIndex >= steps.length) {
-    updateRush();
+    yarnInstall();
```

Comment:
Will this do the yarn.lock update? What happens if there are errors? And can you at least add back a log message explaining what's being done?

Reply:
Yes - good point... wasn't sure if I really want to run this for the user actually for precisely your feedback. I'm thinking of just spitting out a message saying please run yarn yourself kind of thing...